identity/src/main/java/com/android/identity/document/DocumentUtil.kt

```diff
@@ -26,28 +26,28 @@ object DocumentUtil {
     private const val TAG = "DocumentUtil"
 
     /**
-     * A helper for managing a set of credentials.
+     * A helper for managing a set of [SecureAreaBoundCredential]s.
```

Comment:
I still don't think this is specific to only `SecureAreaBoundCredential`, it should work for _any_ `Credential`-derived class.